Source: Imported from gentoo-x86/app-text/sgml-common
Upstream: No
Reason: Bad docdir
--- sgml-common-0.6.3/configure.in
+++ sgml-common-0.6.3/configure.in
@@ -3,7 +3,7 @@ AC_INIT(Makefile.am)
 
 AM_INIT_AUTOMAKE(sgml-common, 0.6.3)
 
-docdir='$(prefix)/doc'
+docdir='$(prefix)/share/doc'
 AC_SUBST(docdir)
 
 dnl Checks for programs.
